.travis.yml

```diff
@@ -42,6 +44,7 @@ install: # Use this to install any prerequisites or dependencies necessary to ru
   # Install dependencies for source repos
   - rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
+  - export PYTHONPATH=/usr/lib/python2.7/dist-packages:$PYTHONPATH
```

Comment:
I confirm that: https://github.com/CentroEPiaggio/kuka-lwr/blob/master/.travis.yml#L2-L4.

In my case, I also dropped the `sudo: required` to engange the container-based build, and it is working too.

Finally, I don't have tests, but you do, at least in the joint_trajectory_controller, so perhaps it'd be useful to run `catkin_make test` in the end, or similar.